### DISCUSSION TITLE
[SUGGESTION] Need option to reverse rotation of rotor for inward looking cameras.

### DISCUSSION THREAD

**Discussion by Servomotor** [tsunamayo/Starship-EVO#4021] [Suggestions]
Good Morning Tsuna. Hope your weekend was well. 

Here is the scenario I am having trouble with.

I have a camera "turret" set up, but the camera is facing **INWARD**, not **OUTWARD** like normal. I want to create a "camera drone" that allows me to fly around my build, but with the camera looking inward towards the build. The setup works, **_except the side to side mouse movement is reversed_** <----this is the core issue). I have to move my mouse left to move the camera right, and vice versa. 

In every turret scenario I can think of, you want the camera facing outward, like the game works now. But in the case of making a standalone camera on a rotor/hinge, it would be nice to have the option to reverse the rotor direction when the camera is facing inward.

I hope this makes sense. You can see in these screenshots how the set up works. I place the camera/hinge at the end of a long stretched set of bricks to get the distance away from the rotor. Then delete those bricks so that they do not interfere with the parent object. 

Think of having a building, with a security camera drone that flies around it, but looking inwards towards the building. 

Ty,
Servo

This picure shows the set up.
![20210208070519_1](https://user-images.githubusercontent.com/65745916/107225306-3f43db80-69de-11eb-8f08-d339258f0627.jpg)

This picture shows me using the yoke and moving the camera around looking inward.
![20210208070535_1](https://user-images.githubusercontent.com/65745916/107225308-3fdc7200-69de-11eb-9dd6-ac1760c4a6a0.jpg)

This picture shows me using the yoke and moving the camera around looking inward.
![20210208070538_1](https://user-images.githubusercontent.com/65745916/107225309-3fdc7200-69de-11eb-81d7-c60b03f179f7.jpg)

This picture shows me using the yoke and moving the camera around looking inward.
![20210208070541_1](https://user-images.githubusercontent.com/65745916/107225312-3fdc7200-69de-11eb-8878-94bf3bb7139c.jpg)

This picture shows me using the yoke and moving the camera around looking inward.
![20210208070544_1](https://user-images.githubusercontent.com/65745916/107225314-40750880-69de-11eb-9512-65c5267ba7ff.jpg)

This picure shows the set up, but with the camera rotated into a new position.
![20210208070621_1](https://user-images.githubusercontent.com/65745916/107225315-40750880-69de-11eb-85d7-84a7f3a60edb.jpg)

**Comment by tsunamayo** [comment]
@Servomotor Hi, thanks mate I hope you had a good one too! I am not sure to understand, so if you could send me your blueprint that would help! Thanks

**Comment by Servomotor** [comment]
I sent you an email with the zipped file and some pictures to help with finding the yoke.
